src/platform/user/profile/vap-svc/README.md

```diff
@@ -11,38 +11,48 @@ Currently, Vet360's scope is limited to veteran contact information, which consi
 - Residential Address
 
 ## Getting started
-The [Profile](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/personalization/profile360) application serves as an example for how to use this work.
+The [Profile](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/personalization/profile) application serves as an example for how to use this work.
```

Comment:
I think it's worth mentioning that the Profile obviously interfaces with VAP, but I think pointing to the Letters app that is using one of the drop-in contact info components would be super useful. I'm still reading through this document, but it seems like it should have two purposes:
1. explain how interaction with VA Profile Service works (the transaction-based nature of it) 
2. explain how to use the drop-in components (that no one but Letters is using right now)